pkg/printers/internalversion/printers.go

```diff
@@ -1857,7 +1860,23 @@ func printStorageClass(obj *storage.StorageClass, options printers.GenerateOptio
 		name += " (default)"
 	}
 	provtype := obj.Provisioner
-	row.Cells = append(row.Cells, name, provtype, translateTimestampSince(obj.CreationTimestamp))
+	reclaimPolicy := string(api.PersistentVolumeReclaimDelete)
+	if obj.ReclaimPolicy != nil {
+		reclaimPolicy = string(*obj.ReclaimPolicy)
+	}
+
+	volumeBindingMode := string(storage.VolumeBindingImmediate)
+	if obj.VolumeBindingMode != nil {
+		volumeBindingMode = string(*obj.VolumeBindingMode)
+	}
+
+	allowVolumeExpansion := false
+	if obj.AllowVolumeExpansion != nil {
+		allowVolumeExpansion = *obj.AllowVolumeExpansion
+	}
+
+	row.Cells = append(row.Cells, name, provtype, reclaimPolicy, volumeBindingMode, allowVolumeExpansion,
+		translateTimestampSince(obj.CreationTimestamp))
```

Comment:
According to our docs, 
default value of ReclaimPolicy is `Delete`
VolumeBindingImmediate is `Immediate`
VolumeBindingImmediate is `false`